R/tdm_utility_functions.R

```diff
@@ -0,0 +1,25 @@
+tdm_scenarios <- function() {
+  c("IPR2021_IPR FPS 2021")
+}
```

Comment:
I don't like this, but this is how it comes out of data_prep, it's a combination of the source and the scenario names. It could be fixed/changed on the data_prep side, but that would mean hard-coding in some adjustment from the raw data, which is probably what's happening with the other scenarios, but since this one is never meant to be seen on the tool or whatever, I figured it was safer to not rock the boat.

Reply:
But yeah, if/when substantial refactoring occurs, this would make sense to adjust.... maybe apply some universal rule to all scenario and source names. Though that could possibly get problematic as well. Honestly, my preference would be to create a fixed lookup table that has our own internal ID for each combination, and then whatever variable names, display names, combo names, whatever could be explicitly defined and controlled in one place.